fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -2421,15 +2421,6 @@ show_param ::=
     {:
         RESULT = new ShowAuthorStmt();
     :}
-    /* Create table */
-    | KW_CREATE KW_TABLE table_name:table
-    {:
-        RESULT = new ShowCreateTableStmt(table);
-    :}
-    | KW_CREATE KW_VIEW table_name:table
-    {:
-        RESULT = new ShowCreateTableStmt(table, true);
-    :}
```

Comment:
why we need to remove this?

Reply:
I guess it is because of the modification of the constructor, the code generated by cup will report an error. I think if we are a complete migration of statements, we can delete the old grammar files

Reply:
> why we need to remove this?

yes, because we have changed the constructor param of the showCreateTableStmt, java cup grammar file still use the old param, and we don't need it any more, so it can be deleted